src/main/java/htsjdk/samtools/cram/io/CramIntArray.java

```diff
@@ -7,7 +7,7 @@
 /**
  * Methods to read and write CRAM array of integers data type.
  */
-public class CramArray {
+public class CramIntArray {
```

Comment:
Renamed because this gave me the impression of a generic array type, though it's limited to integer.  I can revert this if you prefer.